stable/guardian/values.yaml

```diff
@@ -0,0 +1,25 @@
+app:
+  image:
+    repository: odpf/guardian
+    pullPolicy: Always
+    tag: latest
+  container:
+    entryPoint:
+    - serve
+
+  config:
+    LOG_LEVEL: info
+    AUTHENTICATED_USER_HEADER_KEY: x-authenticated-user-email
+    NOTIFIER_PROVIDER: slack
+    JOBS_FETCH_RESOURCES_INTERVAL: '0 */2 * * *'
+    JOBS_REVOKE_EXPIRED_ACCESS_INTERVAL: '*/20 * * * *'
+    JOBS_EXPIRING_ACCESS_NOTIFICATION_INTERVAL: '0 9 * * *'
+
+  secretConfig:
+    ENCRYPTION_SECRET_KEY:
```

Comment:
@rahmatrhd should this encryption secret key be of a particular length, or any length string is okay?